src/index.html

```diff
@@ -4,9 +4,347 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logoImage" class="logo">
+        <img
+          src="images/logo.png"
```

Comment:
smth wrong with bam here
you have logo block (with no styles for logo class) and moyoImg block inside (you styled this block in logo.scss so moyoImg is part of logo according to your logic), just rename moyoImg...